src/main/java/tr/com/srdc/cda2fhir/transform/util/impl/BundleRequest.java

```diff
@@ -56,8 +56,8 @@ public static String generateIfNoneExist(BundleEntryComponent bundleEntry) {
 		// find any overriding OIDs.
 		String[] subsetOIDArray = getDefinedOIDs(bundleEntry.getResource().getResourceType().name());
 
-		if (bundleEntry.getResource().getResourceType().name() != "Medication"
-				&& bundleEntry.getResource().getResourceType().name() != "PractitionerRole") {
+		if (bundleEntry.getResource().getResourceType().name().equals("Medication") == false
```

Comment:
another way of doing this is `if(!bundleEntry.getResource().getResourceType().name().equals("Medication"))`

Reply:
Yeah. I mostly just wanted it to be explicit what the intent of the statement was without having to add comments to the code.